setup.py

```diff
@@ -2,7 +2,7 @@
 import os, sys
 
 setup(name='dailymotion',
-      version='0.2.3',
+      version='0.2.5',
```

Comment:
Seems like the current version of the `dailymotion` package is `0.2.3` (cf. https://pypi.org/project/dailymotion/#history). Do we really want to upgrade it to `0.2.5` and skip the `0.2.4` version?